src/BinaryKits.Zpl.Viewer/ElementDrawers/PDF417ElementDrawer.cs

```diff
@@ -64,16 +73,16 @@ public override void Draw(ZplElementBase element)
                     //{ EncodeHintType.PDF417_AUTO_ECI, true },
                     //{ EncodeHintType.DISABLE_ECI, true },
                     { EncodeHintType.PDF417_COMPACTION, Compaction.AUTO},
-                    { EncodeHintType.PDF417_ASPECT_RATIO, 3 }, // height of a single bar relative to width
-                    { EncodeHintType.PDF417_IMAGE_ASPECT_RATIO, 2.0f }, // zpl default, proportions of columns to rows
+                    { EncodeHintType.PDF417_ASPECT_RATIO, PDF417AspectRatio.AUTO }, // height of a single bar relative to width
+                    { EncodeHintType.PDF417_IMAGE_ASPECT_RATIO, 1.0f }, // zpl default 2.0f, proportions of columns to rows //1.0f looks closer to labelary
```

Comment:
As a competitor to labelary, we shouldn't attempt to emulate them bug for bug. Instead we should match the printers users will be sending their zpl code as closely as possible.

Reply:
I'll print one to see check if there is a difference.

Reply:
It looks like Labelary is right:
<img width="458" alt="image" src="https://github.com/BinaryKits/BinaryKits.Zpl/assets/12891775/08cfb788-1691-47c7-90d6-f8ff6805f521">

The difference is that DHL uses `^B7N,6.7,5,,30,N` for the PDF417 barcode.
`[null],30` is not a valid columns/rows combination and Labelary (and it looks like Zebra) recalculates that into `^B7N,6,5,9,30,N`.

Our PDF417 library starts changing the row count to find a valid match for the columns in: `ZXing.PDF417.Internal.PDF417.determineDimensions`. That results in `^B7N,6,5,9,28`. So that is a limitation that we have to accept for now.

However, while backtesting this I found that I made a mistake with `PDF417_ASPECT_RATIO` and missed the `vertical_scale` function, so I'll prepare some improvements.

Reply:
Thank you for your diligence, as I would not have access to a printer until after tomorrow.